_posts/2017-05-04-amsterdam-genomics.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+subheadline:
+title: "Data Carpentry Genomics comes to the Netherlands"
+teaser: “The Netherlands eScience Center in collaboration with University Medical Center Utrecht, SURFsara and DTL/ELIXIR-NL organized one of the first Data Carpentry Genomics workshops in Europe.
+” 
+header:
+  image_fullwidth: "light-blue-wood-texture.jpg"
+categories:
+  - blog
+comments: true
+show_meta: true
+authors: [Mateusz Kuzak,]
+---
+
+Twenty-five life science researchers from across the Netherlands
+and Germany joined the event on March 22nd and 23rd in Science Park
+Amsterdam. The workshop covered multiple aspects of working with
+genomics data.
+
+Lex Nederbragt from the University of Oslo, kicked off the event
+with an introduction to the core dataset. The rest the first day
+focused on working with R in RStudio environment to explore, clean
+and visualise metadata in a tabular format, presented by Mateusz Kuzak
+from the Netherlands eScience Center and has been wrapped up by Anita
+Schurch from UMC Utrecht introducing the Unix shell and file system
+navigation.
+
+![Lex Nederbragt kicking off the first day]({{ site.urlimg }}/blog/2017-ams/day1.jpeg)
+
+The following day Denis Schmitz and Sam Nooij from National Institute
+for Public Health and the Environment continued the Unix shell lesson,
+showing how to search, organize and document the project and how to
+use for loops to do quality control of sequencing data.  Afterwards,
+Lex dived deeper into shell automation, scripting and workflows.
+Anita followed up with the ways to move data around between the cloud
+and learners' laptops. At the end of this hands-on intensive day,
+Natalie Danezi from SURFsara introduced High-Performance Computing
+resources available to Dutch researchers.
+
+During those two days, participants were not only presented with
+various techniques of efficient data analysis but also had the
+opportunity to learn how to use remote cloud resources provided by
+SURFsara and Amazon. Instructors collaborated with SURFsara
+consultants, Natalie Danezi, Ander Astudillo and Niek Bosch in the
+preparation and documentation of the virtual machine running on
+National HPC Cloud, based on the original Data Carpentry Amazon AMI.
+This image and the documentation will be used in future workshops
+organized within a partnership between ELIXIR and Software and Data
+Carpentry Foundations.
+
+The event closed with drinks and snacks at nearby Maslov Cafe.
+
+![Post workshop recovery session]({{ site.urlimg }}/blog/2017-ams/maslov.jpeg)
+
+We are just at the beginning of the journey. More work needs to be
+done on improving genomics lesson, and Data Carpentry is looking
+forward to contributions from everyone interested in providing help.
+You can read more about it in the recent [Data Carpentry blog post]
+(http://www.datacarpentry.org/blog/genomics-lessons/)
```

Comment:
This link does not render on the webpage, maybe due to a space in between the brackets or so...